Modules/Relationships/src/Relationships.Application/Relationships/Queries/CanEstablishRelationship/CanEstablishRelationshipQuery.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using MediatR;
+
+namespace Backbone.Modules.Relationships.Application.Relationships.Queries.CanEstablishRelationship;
+
+public class CanEstablishRelationshipQuery : IRequest<CanEstablishRelationshipResponse>
+{
+    public required IdentityAddress PeerAddress { get; set; }
```

Comment:
We do not want to use value objects in Mediatr Commands and Queries anymore.
This PR would benefit from https://github.com/nmshd/backbone/pull/757 to add the Validator with the `ValidId` extension method call.
I would say, use `string` and add a `Validator` regardless, as this might be merged before the linked PR.